strings.txt

```diff
@@ -16858,3 +16858,955 @@
+    fr = Pas maintenant
+    da = Ikke nu
+    id = Tidak Sekarang
+    ko = 지금 아님
+    sv = Inte nu
+    tr = Şimdi değil
+    uk = Не зараз
+    vi = Không phải lúc này
+    hu = Most nem
+    de = Nicht jetzt
+    fi = Niet nu
+    cs = Později
+    it = Non ora
+    nb = Ikke nå
+    zh-Hant = 現在不要
+    zh-Hans = 现在不要
+    th = ไม่ใช่ตอนนี้
+    ja = 後で
+    ro = Nu acum
+    ar = ليس الآن
+    sk = Nie teraz
+    es = Ahora no
+    pl = Nie teraz
+    nl = Niet nu
+    pt = Agora não
+    he = לא עכשיו
+
+  [settings_button]
+    en = Settings
+    ru = Настройки
+    fr = Paramètres
+    da = Indstillinger
+    id = Pengaturan
+    ko = 설정
+    sv = Inställningar
+    tr = Ayarlar
+    uk = Налаштування
+    vi = Thiết lập
+    hu = Beállítások
+    de = Einstellungen
+    fi = İnstellingen
+    cs = Nastavení
+    it = Impostazioni
+    nb = Innstillinger
+    zh-Hant = 設置
+    zh-Hans = 设置
+    th = การตั้งค่า
+    ja = 設定
+    ro = Setări
+    ar = الإعدادات
+    sk = Nastavenia
+    es = Ajustes
+    pl = Ustawienia
+    nl = İnstellingen
+    pt = Configurações
+    he = הגדרות
+
+  [placepage_parking_surface]
+    comment = Parking type in Place Page
+    en = Surface
+    ru = Открытая парковка
+    fr = Surface
+    da = Overflade
+    id = Parkir Luar
+    ko = 주차공간
+    sv = Yta
+    tr = Yüzey
+    uk = Відкрите паркування
+    vi = Trên mặt đất
+    hu = Szabadtéri parkoló
+    de = Öffentlicher Parkplatz
+    fi = Pinta
+    cs = Povrch
+    it = Superficie
+    nb = Åpen plass
+    zh-Hans = 表面
+    th = กลางแจ้ง
+    ja = 表面
+    ro = Suprafață
+    ar = سطح
+    sk = Typ parkoviska
+    es = Superficie
+    pl = Powierzchnia
+    nl = Boven de grond
+
+  [placepage_parking_multistorey]
+    en = Multi-storey
+    ru = Многоэтажная наземная парковка
+    fr = Multi-niveaux
+    da = Flere etager
+    id = Bertingkat
+    ko = 주차장 건물
+    sv = Flera våningar
+    tr = Çok Katlı
+    uk = Багатоповерхове наземне паркування
+    vi = Nhiều tầng
+    hu = Parkolóház
+    de = Tiefgarage mit mehreren Etagen
+    fi = Monikerroksinen
+    cs = Vícepatrové
+    it = Su più piani
+    nb = Fleretasjers
+    zh-Hans = 多层
+    th = หลายชั้น
+    ja = 複数階
+    ro = Multi-etajat
+    ar = متعدد الطوابق
+    sk = Viacpodlažné
+    es = Varias plantas
+    pl = Wielopoziomowy
+    nl = Meerlaags
+
+  [placepage_parking_underground]
+    en = Underground
+    ru = Подземная парковка
+    fr = Souterrain
+    da = Underjordisk
+    id = Bawah Tanah
+    ko = 지하
+    sv = Under mark
+    tr = Metro
+    uk = Підземне паркування
+    vi = Dưới mặt đất
+    hu = Föld alatti parkoló
+    de = Tiefgarage
+    fi = Maan alla
+    cs = Podzemní
+    it = Sotterraneo
+    nb = Undergrunns
+    zh-Hans = 地下
+    th = ชั้นใต้ดิน
+    ja = 地下
+    ro = Subteran
+    ar = تحت الارض
+    sk = Podzemné
+    es = Subterráneo
+    pl = Podziemny
+    nl = Onder de grond
+
+  [placepage_parking_rooftop]
+    en = Rooftop
+    ru = Парковка на крыше здания
```

Comment:
Наверное можно, Игорь, что скажешь?

> On Apr 26, 2016, at 4:22 PM, Alex Zolotarev <notifications@github.com> wrote:
> 
> In strings.txt <https://github.com/mapsme/omim/pull/3005#discussion_r61083498>:
> 
> > +    cs = Podzemní
> > +    it = Sotterraneo
> > +    nb = Undergrunns
> > +    zh-Hans = 地下
> > +    th = ชั้นใต้ดิน
> > +    ja = 地下
> > +    ro = Subteran
> > +    ar = تحت الارض
> > +    sk = Podzemné
> > +    es = Subterráneo
> > +    pl = Podziemny
> > +    nl = Onder de grond
> > +
> > +  [placepage_parking_rooftop]
> > +    en = Rooftop
> > +    ru = Парковка на крыше здания
> Можно короче - Парковка на крыше
> И в украинском тоже.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/3005/files/c2b186b3a7f7730ba2f8b359702c3efcb561c295..454ca48abdc1a7a643a178fac869c249f100121f#r61083498>